rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -4,15 +4,16 @@ import cats.syntax.all._
 import cats.effect._
 import cats.effect.concurrent.{Deferred, Ref}
 import coop.rchain.rspace.history.HistoryReaderBase
-import coop.rchain.shared.MapOps._
 import coop.rchain.rspace.internal._
 
 import scala.collection.immutable.Map
 
 trait HotStore[F[_], C, P, A, K] {
-  def getContinuations(channels: Seq[C]): F[Seq[WaitingContinuation[P, K]]]
-  def putContinuation(channels: Seq[C], wc: WaitingContinuation[P, K]): F[Unit]
-  def installContinuation(channels: Seq[C], wc: WaitingContinuation[P, K]): F[Unit]
+  type Continuation = WaitingContinuation[P, K]
```

Comment:
@nzpr, @zsluedem Should we rename class `WaitingContinuation` to `Continuation` or `Kont` or ...?

Reply:
@tgrospic, @nzpr, this type was eliminated in the last implementation.